src/js/components/Box.js

```diff
@@ -26,6 +27,7 @@ var Box = React.createClass({
     ]),
     reverse: React.PropTypes.bool,
     responsive: React.PropTypes.bool,
+    primary: React.PropTypes.bool,
```

Comment:
Good point. I'm not sure where is the best place to put that. But I agree with you. Based on this feedback I believe Article is the best place to put the `Main Content` anchor